### ISSUE TITLE
[Bug] Accessibility issue with footer button colour

### ISSUE THREAD

**Issue by davenichols-DHLS** [hlxsites/danaher-ls-aem#357]
I noticed that our accessibility has gone from 100 to 90, all because of the orange footer button.  Sorry, I should have checked this before.  I am going to amend to fix this.
